pkg/tecdsa/marshaling_test.go

```diff
@@ -51,3 +52,20 @@ func TestPrivateKeyShareMarshalling_NonTECDSAKey(t *testing.T) {
 
 	testutils.AssertErrorsSame(t, ErrIncompatiblePublicKey, err)
 }
+
+func TestSignatureMarshalling(t *testing.T) {
+	signature := &Signature{
+		R:          big.NewInt(100),
+		S:          big.NewInt(200),
+		RecoveryID: 2,
+	}
+
+	unmarshaled := &Signature{}
+
+	if err := pbutils.RoundTrip(signature, unmarshaled); err != nil {
+		t.Fatal(err)
+	}
+	if !reflect.DeepEqual(signature, unmarshaled) {
+		t.Fatal("unexpected content of unmarshaled signature")
+	}
+}
```

Comment:
Should we test the signature marshaling with fuzzer? This would cover `id > math.MaxInt8 || id < math.MinInt8` check in the unmarshaling code.

Reply:
See 4c3a7ec7